dashboard/app/controllers/aichat_controller.rb

```diff
@@ -1,3 +1,7 @@
+require 'cdo/throttle'
+
+AICHAT_PREFIX = "aichat/".freeze
+DEFAULT_REQUEST_LIMIT_PER_MIN = 50
```

Comment:
@sureshc @cat5inthecradle I could use some input on this per-minute, per-user throttling limit. I picked 50 because each AI request is going to take _at least_ 1 second (especially with our new polling/activejob architecture), so if a single user is hitting this limit they're either sending a lot of very small messages during a period of low usage, or potentially have multiple browser tabs open. Also, based on our super rough projections, we expect a maximum overall request rate of ~600/minute for a projected ~7000 users, so again it would be a bit of an anomaly for a single user to hit this rate given what we've seen in our pilot. That being said, are there other considerations I'm missing when setting this limit? Is there a maximum number of queued ActiveJobs for which this limit would need to lower? This is also configurable via DCDO, so we can always update later.